prdoc/pr_5546.prdoc

```diff
@@ -10,12 +10,14 @@ crates:
     bump: patch
   - name: snowbridge-pallet-inbound-queue
     bump: patch
-  - name: snowbridge-pallet-system
+  - name: snowbridge-pallet-outbound-queue
     bump: patch
+  - name: snowbridge-pallet-system
+    bump: major
   - name: snowbridge-core
-    bump: patch
+    bump: major
   - name: snowbridge-router-primitives
-    bump: patch  
+    bump: major
```

Comment:
@claravanstaden Thanks for the fix. Just curious why some of the packages bumped as major?